yb-voyager/cmd/cutoverToTargetCmd.go

```diff
@@ -69,7 +90,7 @@ func init() {
 	BoolVar(cutoverToTargetCmd.Flags(), &prepareForFallBack, "prepare-for-fall-back", false,
 		"prepare for fallback by streaming changes from target DB back to source DB. Not applicable for fall-forward workflow.")
 	BoolVar(cutoverToTargetCmd.Flags(), &useYBgRPCConnector, "use-yb-grpc-connector", true,
-		"Use the gRPC connector for YB export (default: true). If set to false, the logical replication connector (supported in YB versions 2024.1.1+) is used. For this new logical replication based connector, ensure no ALTER TABLE commands causing table rewrites (e.g., adding primary keys) were present in the schema during import")
```

Comment:
I think the help message needs to be much more elaborate. 
We need to help the user make the decision here. We should: 
- probably link to documentation? (which has details about logical vs grpc). 
- either the help msg or docs should inform the user: for grpc - certain ports need to be open and accessible. in situations where ports are not accessible (like YBM), they should use logical connector. 
- logical connector is an alpha/beta release(we still don't have proper testing for this yet)

Reply:
@makalaaneesh I have made this change, can you PTAL again?